src/app/personside/infotabs/oppfolging/OppfolgingDatoKomponent.tsx

```diff
@@ -56,34 +55,47 @@ const tidligsteDato = () =>
     moment()
         .subtract(10, 'year')
         .startOf('day')
-        .toDate();
+        .toISOString()
+        .split('T')[0];
 
 const senesteDato = () =>
     moment()
         .add(1, 'year')
         .endOf('day')
-        .toDate();
+        .toISOString()
+        .split('T')[0];
 
-function getDatoFeilmelding(fra: Date, til: Date) {
-    if (fra > til) {
+function getDatoFeilmelding(fra: string, til: string) {
+    if (Date.parse(fra) > Date.parse(til)) {
```

Comment:
Tror vi skal titte litt mer på denne funksjonen, for jeg føler jeg sitter igjen med en del spørsmål. :D 

1. Hva skjer om `Date.parse` returnerer `Invalid Date` her. 
2. Trenger vi å kalle `Date.parse` 8 ganger for å sjekke to datoer?
3. Trenger vi i det hele tatt `Date.parse`, om input følger ISO8601 så skal jo disse være leksikografisk sorterbare. Så kanskje det holder å sammenligne string-verdiene. Eller kan vi få verdier som ikke følger ISO-standarden?